src/model/position.js

```diff
@@ -493,6 +498,17 @@ export default class Position {
 		return compareArrays( thisParentPath, posParentPath ) == 'same';
 	}
 
+	/**
+	 * Returns a copy of this position that is transformed by given `operation`.
+	 *
+	 * The new position's parameters are updated accordingly to the effect of the `operation`.
+	 *
+	 * For example, if `n` nodes are inserted before the position, the returned position {@link ~Position#offset} will be
+	 * increased by `n`. If the position was in a merged element, it will be accordingly moved to the new element, etc.
+	 *
```

Comment:
I believe that this method is also safe to use it on non-existing positions. I you put such information for the previous method maybe all "safe" methods should get this warning?

Reply:
At least all public methods.